deploy/docs/Installation_with_Helm.md

```diff
@@ -36,66 +41,84 @@ These steps require that no Prometheus exists. If you already have Prometheus in
+### Troubleshooting Installation
+If you get `Error: customresourcedefinitions.apiextensions.k8s.io "alertmanagers.monitoring.coreos.com" already exists` on a Helm2 installation, run the above command with the `--no-crd-hook` flag:
+
+```bash
+helm install sumologic/sumologic --name my-release --set sumologic.accessId=<SUMO_ACCESS_ID> --set sumologic.accessKey=<SUMO_ACCESS_KEY>  --set sumologic.clusterName="<MY_CLUSTER_NAME>" --no-crd-hook
+```
 
-To install on `GKE`, use the provided override file to customize your configuration and uncomment the following lines in the `values.yaml` file referenced below:
+If `helm upgrade --install` hangs, it usually means the pre-install setup job is failing and is in a retry loop. Due to a Helm limitation, errors from the setup job cannot be fed back to the `helm upgrade --install` command. Kubernetes schedules the job in a pod, so you can look at logs from the pod to see why the job is failing. First find the pod name in the namespace where the Helm chart is deployed:
+```sh
+kubectl get pods -n sumologic
+```
 
+Get the logs from that pod:
 ```
-ebpf:
-  enabled: true
+kubectl logs POD_NAME -f
 ```
 
-To customize your configuration:
+If you get `Error: collector with name 'sumologic' does not exist
+sumologic_http_source.default_metrics_source: Importing from ID`, you can safely ignore it and the installation should complete successfully. The installation process creates new [HTTP endpoints](https://help.sumologic.com/03Send-Data/Sources/02Sources-for-Hosted-Collectors/HTTP-Source) in your Sumo Logic account, that are used to send data to Sumo. This error occurs if the endpoints had already been created by an earlier run of the installation process.
 
-- Download the `values.yaml` file by running:
+You can find more information in our [troubleshooting documentation](Troubleshoot_Collection.md).
 
+### Customizing Installation
+All default properties for the Helm chart can be found in our [documentation](HelmChartConfigiuration.md). We recommend creating a new `values.yaml` for each Kubernetes cluster you wish to install collection on and setting only the properties you wish to override. Once you have customized you can use the following commands to install or upgrade. 
+  
+Helm2
 ```bash
-curl https://raw.githubusercontent.com/SumoLogic/sumologic-kubernetes-collection/release-v0.17/deploy/helm/sumologic/values.yaml > values.yaml
+helm upgrade --install sumologic/sumologic --name my-release -f values.yaml
```

Comment:
This should work as well:
`helm upgrade --install my-release sumologic/sumologic -f values.yaml`

It's the same we use for Helm 3 below.

Reply:
Good call...will update